docs/documentation/cl_options/index.rst

```diff
@@ -0,0 +1,40 @@
+Command Line Arguments
+======================
+
+What starting vimiv form the command line you have the ability to pass a number of
+different argument to vimiv. The general calling structure is depicted below:
+
+.. include:: synopsis.rstsrc
+
+Command Line Arguments
+----------------------
+
+The following is an exhaustive list of all available arguments:
+
+.. include:: options.rstsrc
+
+Examples
+--------
+
+In the following we present a few use cases of command line arguments.
+
+* Start in thumbnail view instead of image view
+    ``vimiv * --command "enter thumbnail" --command "enter library"``
+
+* Start in read-only mode. This prevents accidental modification (renaming, moving, editing etc.) of any images
+    ``vimiv --set read_only true``
+
+* Change WM_CLASS_INSTANCE to identify a vimiv instance
+    ``vimiv --qt-args '--name myVimivInstance'``
```

Comment:
Nit-picking, but you use double-quotes everywhere else, so I would also use them here :blush:

Reply:
Good catch. That's caused by me copy/pasting examples from different issues :upside_down_face: 

fixed in e6a562414448933601ee2b9b8f0000c9fe266fc2